internal/strdist/strdist.go

```diff
@@ -28,6 +28,28 @@ func StandardCost(ar, br rune) Cost {
 	return Cost{SwapAB: 1, DeleteA: 1, InsertB: 1}
 }
 
+// StandardGlobCost encodes the standard Levenshtein distance with support for
+// "*", "?" and "**". However, because it works on runes "**" has to be encoded
+// as "⁑" in the strings.
+//
+// Supported wildcards:
+//
+//	?  - Any one character
+//	*  - Any zero or more characters
+//	⁑  - Any zero or more characters
+func StandardGlobCost(ar, br rune) Cost {
```

Comment:
And yet, this standard is not what we use in GlobPath.  Any reason why we're not just using GlobPath for our match?  We won't have slashes in those strings, so it won't affect anything.

Reply:
I was not comfortable having some logic that was treating "/" differently. For example: python3.1x matches python3.10 where x is any character **except** when that character is a "/".

Reply:
How is a slash supposed to be seen at all, in either of these strings?  These are package and slice names.

In either case, the "Standard" in the StandardCost function comes from the fact that this is how one would get a classic Levenshtein distance from the books.  There's nothing standard about this function. It's not even what we use ourselves in our own glob path matching.